evergreen.yml

```diff
@@ -1495,7 +1447,9 @@ functions:
           export RUST_BACKTRACE=1
           # Start a local ADF
           $COMMON_TEST_INFRA_DIR/test-environment/run_adf.sh start &&
-          cargo run --bin data_loader &&
+          cd $COMMON_TEST_INFRA_DIR &&
+          cargo run --bin data-loader -- --mongod-uri mongodb://localhost:28017 --adf-uri mongodb://${adf_test_local_user}:${adf_test_local_pwd}@localhost:27017 -d ../resources/integration_test/testdata &&
```

Comment:
As part of the refactor ticket I'll file, or as part of one of the env variable tickets backlogged in the epic, we can update this to use the env variables instead of command line args. The data-loader is already set up to use environment variables but I didn't want to spend time setting up a parallel set of names just for this. For now cli args are sufficient.